verification/store_test.go

```diff
@@ -16,36 +19,61 @@ func TestLoadValidTrustStore(t *testing.T) {
 }
 
 func TestLoadSymlinkTrustStore(t *testing.T) {
-	_, err := LoadX509TrustStore("testdata/trust-store/valid-trust-store_SYMLINK")
-	if err == nil || err.Error() != "\"testdata/trust-store/valid-trust-store_SYMLINK\" is not a regular directory (symlinks are not supported)" {
+	// TODO run symlink tests on Windows
+	if runtime.GOOS == "windows" {
```

Comment:
Skipping symlink tests on Windows as it requires Admin permissions to create symlinks. [source](https://github.com/golang/go/issues/22874)